examples/cmd/run.py

```diff
@@ -26,20 +27,37 @@ def save_wav_file(wav, index):
 
 def main():
     # Retrieve text from command line argument
+    try:
+        sys.argv.remove('--stream')
+        stream = True
+    except:
+        stream = False
     text_input = sys.argv[1] if len(sys.argv) > 1 else "<YOUR TEXT HERE>"
     print("Received text input:", text_input)
 
     chat = ChatTTS.Chat()
     print("Initializing ChatTTS...")
+    # if using macbook(M1), I suggest you set `device='cpu', compile=False`
```

Comment:
You can give more reason of why not use mps.

Reply:
Well, I just had a test and you are right. I will try to figure out why mps is even slower.